dawn/test/unit-test/dawn-c/TestCompiler.cpp

```diff
@@ -38,7 +38,9 @@ static void freeCharArray(char** array, int size) {
   std::free(array);
 }
 
-TEST(CompilerTest, CompileEmptySIR) {
+// TODO empty is not legal anymore as gridtype is required (the default is illegal)
```

Comment:
@review Not sure what to do here. Delete the test? Or change it to a minimal SIR with only required info.

The reason this doesn't work anymore is that we removed default values in protobuf enums.

Reply:
Does compiling an SIR with no Stencils break things? That would be my initial approach here

Reply:
SIR with no stencils should work. Now the question is: do we still start from an sir string? It seems too verbose to write the string in code, so maybe deserializing sir from file is the way?

Reply:
I tried to make it work until I realized the `dawnCompile` expects a byte encoded protobuf message. Not sure if it makes sense to embed byte data.

Reply:
understood. So should we drop it?

Reply:
I deleted it.